From 8a266d545c20eecece1d26dd41550cab227928c7 Mon Sep 17 00:00:00 2001
From: Hannah von Reth <hannah.vonreth@owncloud.com>
Date: Mon, 4 Nov 2019 09:06:48 +0100
Subject: [PATCH] Fix mac build

---
 CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 18fadd15..cb17e499 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -3,6 +3,8 @@ cmake_minimum_required(VERSION 3.12)
 add_definitions(-DNO_STABS_SUPPORT)
 
 if(APPLE)
+add_definitions(-DHAVE_MACH_O_NLIST_H)
+
 add_executable(dump_syms
     src/tools/mac/dump_syms/dump_syms_tool.cc
     src/common/mac/macho_walker.cc
@@ -19,6 +21,8 @@ add_executable(dump_syms
     src/common/language.cc
     src/common/dwarf_line_to_module.cc
     src/common/path_helper.cc
+    src/common/stabs_reader.cc
+    src/common/stabs_to_module.cc
     src/common/dwarf_cfi_to_module.cc
     src/common/dwarf_cu_to_module.cc
     src/common/dwarf/functioninfo.cc
