diff -Nrub -x '*~' -x '*\.rej' -x '*\.orig' -x '*\.o' -x '*\.pyc' -x CMakeLists.txt.user dpkg-sig-0.13.1+nmu2.orig/dpkg-sig dpkg-sig-0.13.1+nmu2/dpkg-sig
--- dpkg-sig-0.13.1+nmu2.orig/dpkg-sig	2016-12-19 23:02:40.000000000 +0000
+++ dpkg-sig-0.13.1+nmu2/dpkg-sig	2023-08-03 10:25:03.302115192 +0000
@@ -1,4 +1,5 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
+use warnings;
 #
 # dpkg-sig signs deb-files in a standard way
 #
