From aca4121d4ccca0201b58cbb7c278a6be2b9cdd72 Mon Sep 17 00:00:00 2001
From: Hannah von Reth <vonreth@kde.org>
Date: Fri, 28 Apr 2023 14:05:31 +0200
Subject: [PATCH] Qt6 support in find_dependency

---
 src/KDSingleApplicationConfig.cmake.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/KDSingleApplicationConfig.cmake.in b/src/KDSingleApplicationConfig.cmake.in
index d58061e..453f024 100644
--- a/src/KDSingleApplicationConfig.cmake.in
+++ b/src/KDSingleApplicationConfig.cmake.in
@@ -8,8 +8,8 @@
 #
 include(CMakeFindDependencyMacro)
 
-find_dependency(Qt5Widgets REQUIRED)
-find_dependency(Qt5Network REQUIRED)
+find_dependency(Qt@QT_VERSION_MAJOR@Widgets REQUIRED)
+find_dependency(Qt@QT_VERSION_MAJOR@Network REQUIRED)
 
 # Add the targets file
 include("${CMAKE_CURRENT_LIST_DIR}/KDSingleApplicationTargets.cmake")
